descriptors/ModuleDescriptor-template.json

```
@@ -2,6 +2,18 @@
   "id": "${artifactId}-${version}",
   "name": "Inventory Module",
   "provides": [
+    {
+      "id": "inventory-config",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/inventory/config/instances/blockedFields",
```

Personally, I prefer hyphenating rather than camel casing in resource paths

```suggestion
          "pathPattern": "/inventory/config/instances/blocked-fields",
```